app/views/produce_items/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Produce</h1>
+
+<table>
+    <% @produce_items.each do |produce|  %>
```

Comment:
```suggestion
    <% @produce_items.find_each do |produce|  %>
```

`find_each` serves the same purpose as `each`, but does not eagerly load all the records. Since this is an easy optimization fix, I think we should apply it.

More on find_each [here](https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each)

Reply:
Now my view test is failling because I can't use the find_each on an array. I'm trying to figure out how to rewrite it, but so far no luck. How could I rewrite it, should we keep this test anyway?

Reply:
You should assign an ActiveRecord relation instead of an array, since this is what will happen in the real application. So you'd do something like:

```rb
ProduceItem.create!(...)
ProduceItem.create!(...)
assign(:produce_items, ProduceItem.all)
```